### PR TITLE
#5 oprava 2 chyb

### PR DESCRIPTION
**Pull request by viktorria**

### CODE REVIEW

File: hanoi.c [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 }
 
 int main()
 {
   int n = 4;
 
-  hanoi(n, 1, 2);
+  hanoi(n, 1, 2, 3);
   return 0;
 }
```
Comment (ZoltanOnody) [inline hanoi.c]: You are missing the endline!!!!

File: hanoi.c [hunk 2]
```diff
@@ -1,21 +1,21 @@
 #include <stdio.h>
 
-void hanoi(int n, int a, int b)
+void hanoi(int n, int a, int b, int c)
 {
   if (n == 1)
```
Comment (ZoltanOnody) [inline hanoi.c]: Are you sure about this condition? Shouldn't it be `n== -1`?

Comment (nitrajka) [reply hanoi.c]: no

Comment (ZoltanOnody) [reply hanoi.c]: why? could you clarify your statement?

File: hanoi.c (hunk 1 above)
Comment (viktorria) [reply hanoi.c]: just a detail, sorry :(

**Comment by Radcho** [comment]
This fixes a lot more than just two mistakes...

closes #5

File: hanoi.c (hunk 2 above)
Comment (viktorria) [reply hanoi.c]: Zoltan told me. Ask him.

Comment (ZoltanOnody) [reply hanoi.c]: Yeah, I was wrong sorry about it. `n == 1` is alright.

**Pull request merged**